docs/sql/system.txt

```diff
@@ -1100,6 +1100,14 @@ The mapping follows the following pattern:
 | ``my_schema.parted_table`` (value=null) | ``my_schema..partitioned.my_table.0400`` |
 +-----------------------------------------+------------------------------------------+
 
+.. note::
+
+    Snapshot/Restore is very similar to replication and operates on a per-shard basis.
+    Hence, the state indicates whether all (``SUCCESS``), some (``PARTIAL``),
+    or no shards(``FAILED``) have been backed up. In case of a ``PARTIAL`` state
+    some primaries became unavailable and *another snapshot should be created*!
```

Comment:
pls descibe more detailed what it means if primaries shards became unavailable and why another snapshot is needed (something like not all records were backed up and so rerun until a success state is required in order to restore all records)